20230727/s9-1690470790793/README.md

```diff
@@ -0,0 +1,66 @@
+# Scenario 8 - ALS Baseline with Sims-only, multiple k6 VUs
```

Comment:
Is this the right description for Scenario 9?

Reply:
Fixed.